### ISSUE TITLE
Regarding the max_train_steps parameter and volume gs loss

### ISSUE THREAD

**Issue by dzp62442** [WU-CVGL/OmniScene#29]
Hello! During debugging, I noticed that the `max_train_steps` parameter in the configuration file is set to 5000. This causes the loss terms for volume gs to no longer be calculated once the iteration exceeds 4000 (`iter_end - 1000`). Could you explain the rationale behind this design choice?

**Comment by wswdx** [comment]
> Hello! During debugging, I noticed that the `max_train_steps` parameter in the configuration file is set to 5000. This causes the loss terms for volume gs to no longer be calculated once the iteration exceeds 4000 (`iter_end - 1000`). Could you explain the rationale behind this design choice?

Hi. We did this because we found that, when the volume branch was already well-trained before the last few steps of training, we can achieve better visual quality by using unified losses without specific losses for different branches.

**Comment by dzp62442** [comment]
Thank you for your reply! 
I now understand why the loss terms for volume gs are stopped early. However, why is the max_train_steps parameter in the configuration file set to 5000? It seems more reasonable to set it to 100,000.

**Comment by wswdx** [comment]
> Thank you for your reply! I now understand why the loss terms for volume gs are stopped early. However, why is the max_train_steps parameter in the configuration file set to 5000? It seems more reasonable to set it to 100,000.

Sorry for the confusion. It was a mistake. We'll fix it. However, in the code, the maximum training steps is determined by `max_epochs` in the config file.

**Comment by Gxt2002** [comment]
> > 谢谢你的回复！我现在明白为什么体积g的损耗项会提前停止。但是，为什么配置文件中的max_train_steps参数被设置为5000？设定为10万似乎更合理。
> 
> 抱歉让你困惑了。那是个错误。我们会解决的。然而，在代码中，最大训练步数由配置文件中确定。`max_epochs`

However, the configuration file 5000 did indeed set the volume to stop calculating the loss, and the visualization result of the volume is also a mess. How should I set it up correctly?
